dashboard/test/lib/school_info_deduplicator_test.rb

```diff
@@ -139,4 +166,15 @@ class MockSchoolInfoDeduplicator
     assert deduped, "Expected to dedupe on school id"
     assert_equal school_info.id, mock.school_info.id
   end
+
+  test 'missing or blank fields do not match none-null values' do
+    mock = MockSchoolInfoDeduplicator.new
```

Comment:
nit: I see this is following an existing pattern, so no need to change, but I would prefer naming this variable something more contextual and descriptive like `deduplicator` or `mockDeduplicator`.  The variable name `mock` is unhelpful, like calling a variable `instance` or `data`.

Reply:
I went ahead and renamed this.